test/test_notebook.py

```diff
@@ -119,6 +119,21 @@ def test_agentchat_cost_token_tracking(save=False):
     run_notebook("agentchat_cost_token_tracking.ipynb", save=save)
 
 
+@pytest.mark.skipif(
+    skip or not sys.version.startswith("3.10"),
+    reason="do not run if openai is not installed or py!=3.10",
+)
+def test_agentchat_groupchat_customized(save=False):
+    run_notebook("agentchat_groupchat_customized.ipynb", save=save)
+
+@pytest.mark.skipif(
+    skip or not sys.version.startswith("3.10"),
+    reason="do not run if openai is not installed or py!=3.10",
+)
+def test_agentchat_groupchat_stateflow(save=False):
+    run_notebook("agentchat_groupchat_stateflow.ipynb", save=save)

```

Comment:
Adding both to py3.10 will make the test workload imbalanced.

Reply:
Got it!